builtin/staker/stakes/stakes.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2025 The VeChainThor developers
+//
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package stakes
+
+import "math/big"
+
+type WeightedStake struct {
+	vet        *big.Int // The amount of VET staked
+	Multiplier uint8    // The multiplier applied to the stake
+}
```

Comment:
```suggestion
type WeightedStake struct {
	Stake        *big.Int // The amount of VET staked
	Weight  *big.Int    // The multiplier applied to the stake
}
```

To make it work with `delta.Renewal`